unfoldingWord/translationCore
# Added the UDB to the Scripture Pane

#### This pull request addresses:

Write a few sentences about what your pull request addresses, the relevant issue numbers, etc.
This simply adds the UDB to our static folder and then allows the user to select it in the scripture pane


#### How to test this pull request:

Include testing instructions so that other developers know what to look for, and how to make sure that your feature/bugfix/enhancement really works.
1. Load a project
2. See if the UDB is there
3. Try deleting it, and adding it back


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/830)
